assignment/src/com/assignment3/nullException/Main.java

```diff
@@ -0,0 +1,12 @@
+package com.company;
+
+public class Main {
+
+    public static void main(String[] args) {
+        //main class object
+        ExceptionHandling obj=new ExceptionHandling();
+        //assigning the methods
+        obj.divideByZero();
+        obj.nullPointer();
+    }
```

Comment:
Need to multiple exception handling within one method for both exception

Reply:
fix